lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -33,9 +33,10 @@ function multiplyFractions(node) {
   const numeratorArgs = [];
   const denominatorArgs = [];
   node.args.forEach(operand => {
-    if (Node.Type.isOperator(operand, '/')) {
-      numeratorArgs.push(operand.args[0]);
-      denominatorArgs.push(operand.args[1]);
+    if (Node.Type.isFraction(operand)) {
+      const fraction = Node.Type.getFraction(operand);
```

Comment:
the getFraction will remove the unaryminus which breaks stuff - you probably want to move that unary minus into the numerator (I feel like there's code somewhere that does this already but I don't remember where 

maybe in flatten? It'd be nice to have it there to always turn -(a/b) into -a / b then you won't have to deal with it here 

what do you think?